core/templates/dev/head/components/ck-editor-helpers/ck-editor-4-rte.directive.ts

```diff
@@ -36,16 +36,21 @@ angular.module('oppia').directive('ckEditor4Rte', [
         var _RICH_TEXT_COMPONENTS = RteHelperService.getRichTextComponents();
         var names = [];
         var icons = [];
+        var isLessonRelatedContext = (
```

Comment:
Variable names should not start with "is". Perhaps contextIsLessonRelated?

Reply:
Done